releasenotes/notes/enable-transport-for-metadata-calls-727408e9f5916cb7.yaml

```
@@ -0,0 +1,14 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Use the default agent transport for host metadata calls.
+    This allows usage of the config `no_proxy` setting for host metadata calls.
+    By default metadata IP's are added to the transport's `no_proxy` list.
```

I think we should make it clear that we're talking about cloud metadata and not datadog:

```suggestion
    By default cloud provider metadata IP's are added to the transport's `no_proxy` list.
```

WDYT ?